src/index.html

```diff
@@ -22,6 +30,367 @@
   </head>
 
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <div class="header__icon">
+        <a href="#home">
+          <img
+            src="./images/logo.png"
+            alt="Moyo Icon"
+            width="40"
+            height="40"
+          />
+        </a>
+      </div>
+      <nav class="header__nav">
+        <ul class="header__list">
+          <li class="header__item">
+            <a
+              class="header__link is-active"
+              href="#apple"
+            >
+              Apple
+            </a>
+          </li>
+          <li class="header__item">
+            <a
+              class="header__link"
+              href="#samsung"
+            >
+              Samsung
+            </a>
+          </li>
+          <li class="header__item">
+            <a
+              class="header__link"
+              href="#smartphones"
+            >
+              SMARTPHONES
+            </a>
+          </li>
+          <li
+            data-qa="hover"
+            class="header__item"
+          >
+            <a
+              data-qa="nav-hover"
+              class="header__link"
+              href="#laptops-computers"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+          <li class="header__item">
+            <a
+              class="header__link"
+              href="#gadgets"
+            >
+              Gadgets
+            </a>
+          </li>
+          <li class="header__item">
+            <a
+              class="header__link"
+              href="#tablets"
+            >
+              Tablets
+            </a>
+          </li>
+          <li class="header__item">
+            <a
+              class="header__link"
+              href="#photo"
+            >
+              Photo
+            </a>
+          </li>
+          <li class="header__item">
+            <a
+              class="header__link"
+              href="#video"
+            >
+              Video
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="main">
+      <div
+        class="card"
+        data-qa="card"
+      >
+        <img
+          class="card__image"
+          src="./images/imac.jpeg"
+          alt="imac"
+        />
+        <div class="card__content">
+          <div class="card__title">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </div>
+          <div class="card__code">Product code: 195434</div>
+          <div class="card__feedback">
+            <div class="card__stars">
+              <div class="card__star"></div>
+              <div class="card__star"></div>
+              <div class="card__star"></div>
+              <div class="card__star"></div>
+              <div class="card__star card__star--unfilled"></div>
```

Comment:
The star rating system should be implemented in a way that it can be reused and its value easily changed. Consider creating a separate component with a variable that controls the number of active stars.